packages/utils/telemetry-utils/src/staging.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ILoggingError } from "@fluidframework/common-definitions";
+
+// ///////////////////////////////////////////////////////////////////////// //
+//        THIS CODE TO BE MOVED TO COMMON-DEFINITIONS AND COMMON-UTILS       //
+// ///////////////////////////////////////////////////////////////////////// //
+
+/**
+ * All normalized errors flowing through the Fluid Framework adhere to this readonly interface.
+ * It includes Error's properties but as optional, plus errorType and fluidErrorCode strings.
+ */
+export interface IFluidErrorBase extends Readonly<Partial<Error>> {
+    readonly errorType: string;
```

Comment:
Do we support IFluidErrorBase that is not isILoggingError?
Do we need to?

Reply:
Yes, I'm planning to update `assert` in `common-utils` to emit `IFluidErrorBase` but don't want to implement `getTelemetryProperties`, and don't need to due to `mixinTelemetryProps`.

Reply:
If it reaches logging code unchanged, it should not log anything as this object should be considered "foreign", as it is not ILoggingError, right? So the assumption is that someone needs to wrap it.

I personally think we should have one clean requirement about "in" and "out" when it comes to trusting any property to be recorded in telemetry. I do not see how we can differentiate an assert example from some 3rd party using same errorType property name and sticking PII into it.

Reply:
We all agree that wrapping should only happen at a clear abstraction layer - otherwise preserve the original error that was thrown.  There's no need for the decision to wrap to have anything to do with whether we want to annotate an error we encounter for better logging - we should be and can be free to annotate an assert or foreign error without wrapping it.

See issue #6485 and PR #6720 - We will no longer make any assumption about logging the actual properties off an error, it should all go through `getTelemetryProperties` - and even for `LoggingError` they will be stored in a property bag not read directly off the object.